README.md

```diff
@@ -109,11 +120,33 @@ $ TAG=v0.0.2 make tag
 $ GITHUB_TOKEN=xxx make clean release
 ```
 
-### Testing
+
+## Identify the service account from the google API json key file
+### Use the --googleAPIjsonkeypath argument to indicate the keyfile path (note: the --googleProjectID Id will be extracted from the specified file )
 
 ```
 go test -v -cover ./... -args \
   --googleAPIjsonkeypath=XXX \
   --googleAPIdatasetID=XXX \
   --googleAPItableID=XXX \
 ```
+
+
+## Identify the service account from env var GOOGLE_APPLICATION_CREDENTIALS via Google Application Default Credentials ([ADC](https://cloud.google.com/docs/authentication/production#automatically))
+### Use the --googleProjectID argument to indicate GCP ProjectId and let ADC identify the service account from the GOOGLE_APPLICATION_CREDENTIALS env var
+```
+GOOGLE_APPLICATION_CREDENTIALS=../../private.key.json \
+go test -v -cover ./... -args \
+  --googleProjectID=XXX \
+  --googleAPIdatasetID=XXX \
+  --googleAPItableID=XXX \
+```
+
+## Identify the service account via Google Application Default Credentials ([ADC](https://cloud.google.com/docs/authentication/production#automatically))
+### Use the --googleProjectID argument to indicate GCP ProjectId and let ADC identify the default service account
```

Comment:
```suggestion
### Credentials via default service account

Identify the service account via Google Application Default Credentials ([ADC](https://cloud.google.com/docs/authentication/production#automatically)).

Use the `--googleProjectID` argument to indicate GCP Project ID and let ADC identify the default service account. This will only work if you're running on Google's infrastructure (GCP VMs, GKE, etc.).

```